daemon/mgr/container_utils_test.go

```diff
@@ -1,214 +0,0 @@
-package mgr
```

Comment:
Why this file is deleted?

Reply:
This is intended. This file will cause potential CI fails.